README.md

```diff
@@ -14,10 +14,8 @@ To see a plain text representation, add `?debug` to the end of the URL:
 * vehiclePositionsUrl = `http://localhost:8088/vehicle-positions?debug`
 
 To run: 
-
-`java -jar cutr-gtfs-realtime-bullrunner-0.9.0-SNAPSHOT.jar  --tripUpdatesUrl=http://localhost:8080/trip-updates   --vehiclePositionsUrl=http://localhost:8080/vehicle-positions`
-
-...from TARGET directory
-
-The Bull Runner GTFS can be found [here](https://github.com/CUTR-at-USF/bullrunner-gtfs-realtime-generator/blob/master/bullrunner-gtfs.zip) and should be extracted into `../myGTFS/`, as the GTFS-rt feed requires it to run.
+1. Create key.txt file in the main directory and save the API key in the file (API key can be requested from [Syncromatics](http://gmvsyncromatics.com/contact/))
+2. Extract the bullrunner-gtfs.zip file in the same folder. The extracted folder should be name "bullrunner-gtfs"
+3. Build `mvn package`
+4. Run `cd target; java -jar cutr-gtfs-realtime-bullrunner-1.0.0-SNAPSHOT.jar  --tripUpdatesUrl=http://localhost:8080/trip-updates   --vehiclePositionsUrl=http://localhost:8080/vehicle-positions`
```

Comment:
Let's change this port numbers in 4) to `8088` to match the port number shown in the example URLs.